tools/cldr-code/src/main/java/org/unicode/cldr/util/VettingViewer.java

```diff
@@ -549,33 +524,23 @@ public void addAll(FileInfo other) {
             warningSubtypeCounter.addAll(other.warningSubtypeCounter);
         }
 
-        private FileInfo getFileInfo(CLDRFile sourceFile, CLDRFile baselineFile,
-            Relation<R2<SectionId, PageId>, WritingInfo> sorted,
-            EnumSet<Choice> choices, String localeID,
-            T user, Level usersLevel, boolean quick) {
-            return this.getFileInfo(sourceFile, baselineFile, sorted,
-                choices, localeID,
-                user, usersLevel, quick, null);
-        }
-
         /**
          * Loop through paths for the Dashboard or the Priority Items Summary
          *
          * @param sourceFile
-         * @param baselineFile
+         * @param baselineFile null (unused) for Dashboard; maybe used for Priority Items Summary
          * @param sorted
          * @param choices
          * @param localeID
          * @param user
          * @param usersLevel
-         * @param quick
-         * @param xpath
-         * @return
+         * @param specificSinglePath if not null, skip all paths except this one
+         * @return the FileInfo
          */
         private FileInfo getFileInfo(CLDRFile sourceFile, CLDRFile baselineFile,
             Relation<R2<SectionId, PageId>, WritingInfo> sorted,
             EnumSet<Choice> choices, String localeID,
-            T user, Level usersLevel, boolean quick, String xpath) {
+            T user, Level usersLevel, String specificSinglePath) {
```

Comment:
"quick" was unused (always false); "xpath" and "path" were too easily confused with each other, so "xpath" is descriptively renamed "specificSinglePath"